backend/src/API/teamEventsAPI.ts

```diff
@@ -48,6 +48,12 @@ export const createTeamEvent = async (
   const canCreateTeamEvent = await PermissionsManager.canEditTeamEvent(user);
   if (!canCreateTeamEvent)
     throw new PermissionError(`User with email ${user.email} cannot create team events`);
+  if (Number(teamEventInfo.numCredits) > Number(teamEventInfo.maxCredits))
+    throw new BadRequestError('The maximum credits cannot be greater than the team event credit!');
+  if (Number(teamEventInfo.maxCredits) % Number(teamEventInfo.numCredits) !== 0)
+    throw new BadRequestError(
+      'The maximum credits needs to be a multiple of the team event credit!'
+    );
```

Comment:
could be nice to pull out a shared helper for this between this func and `updateTeamEvent`